nautobot/core/settings.py

```diff
@@ -225,7 +225,7 @@
         "social_core.pipeline.social_auth.associate_user",
         "social_core.pipeline.social_auth.load_extra_data",
         "social_core.pipeline.user.user_details",
-        "nautobot.extras.group_sync",
+        "nautobot.extras.group_sync.group_sync",
```

Comment:
Looks like we missed making a similar correction to `nautobot/docs/user-guide/administration/configuration/authentication/sso.md` - would be good to include in the release.

Reply:
I see #6636 is addressing this.